Add Bright Text and Configurable Row Opacity Options

darkharasho opened the pull request:
## Pull Request: Add Configurable Header and Row Styles (Opacity & Color)
<img width="405" height="362" alt="image" src="https://github.com/user-attachments/assets/ea96d57e-2b5f-4418-a8bb-401041706517" />


### What's Changed

- **New Config Options:**
  - `headerStyle`: Object with `opacity` and `color` properties to control the table header's appearance.
  - `rowStyle`: Object with `opacity` and `color` properties to control the table rows' appearance.

- **Usage Example:**
    ```js
    config: {
        headerStyle: { opacity: 1, color: "#fff" },
        rowStyle: { opacity: 0.85, color: "#eee" }
    }
    ```

- **Behavior:**
  - If `opacity` or `color` is set to `null`, that style is not applied.
  - Styles are applied inline to ensure they are respected regardless of MagicMirror theme.

- **README Updated:**  
  - Documentation now reflects the new nested style options and provides usage examples.

---

**These changes give users fine-grained control over the look of their scraped tables, improving both readability and integration with custom MagicMirror themes.**

AndyHazz commented:
I've checked this out - brightText is a nice addition and works fine, but there's something that feels a bit counter-intuitive with a `rowOpacity` value that only serves to darken the text when `brightText` is `true` :thinking: as most people would have the text over the default black background so you're brightening text only to darken it again.

You could probably remove the condition requiring  `brightText` = `true` so it's a bit more straightforward - I agree they're best used in combination, but there's probably no reason to restrict that?

Also not sure why you've set the default to 0.92, and not 1? I'd suggest setting the default to 1, and have the condition that the inline opacity style isn't applied if `rowOpacity` = `1`.

darkharasho commented:
that is a good point, I could remove brightText and just have the default row opacity be the dimmer version. The only thing there that would be tricky is the header.

maybe i just have an opacity option for both row and header. does that sound cleaner to you?

AndyHazz commented:
It seemed like your original intention was to have the brightest/whitest text possible, but also with some (optional) transparency - so there's still benefit to having separate brightness and opacity?

Another way to approach this could be to have `rowColor` and `rowOpacity`? You could then set rowColor to any CSS value e.g. `white` or `#FFF`. That'd achieve the same as `brightText` but with more flexibility. If the default is undefined it'd set no inline styles, so anyone can still use their own custom CSS more easily.

darkharasho commented:
@AndyHazz i implemented what i had in mind + your feedback and updated the pr description to match the new settings. please take a look and see if that makes more sense to you, i think it looks better personally
